dynamodb/dynamo_lock_table_test.go

```diff
@@ -81,7 +81,52 @@ func TestCreateLockTableIfNecessaryTableAlreadyExists(t *testing.T) {
 		assertCanWriteToTable(t, tableName, client)
 
 		// Try to create the table the second time and make sure you get no errors
-		err := CreateLockTableIfNecessary(tableName, client, mockOptions)
+		err = CreateLockTableIfNecessary(tableName, nil, client, mockOptions)
 		assert.Nil(t, err, "Unexpected error: %v", err)
 	})
 }
+
+func TestTableTagging(t *testing.T) {
+	t.Parallel()
+
+	mockOptions, err := options.NewTerragruntOptionsForTest("dynamo_lock_test_utils")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var tags = make(map[string]string)
+	tags["team"] = "team A"
+
+	// Create the table the first time
+	withLockTableTagged(t, tags, func(tableName string, client *dynamodb.DynamoDB) {
+		assertCanWriteToTable(t, tableName, client)
+
+		assertTags(tags, tableName, client, t)
+
+		// Try to create the table the second time and make sure you get no errors
+		err = CreateLockTableIfNecessary(tableName, nil, client, mockOptions)
+		assert.Nil(t, err, "Unexpected error: %v", err)
+	})
+}
+
+func assertTags(expectedTags map[string]string, tableName string, client *dynamodb.DynamoDB, t *testing.T) {
+	var description, err = client.DescribeTable(&dynamodb.DescribeTableInput{TableName: &tableName})
```

Comment:
Idiomatic approach is `aws.String(tableName)` instead of `&tableName`.

Reply:
Done